.github/workflows/update-israeli-bank-scrapers.yml

```diff
@@ -0,0 +1,77 @@
+name: Update israeli-bank-scrapers
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  # Manual trigger
+  workflow_dispatch:
+    inputs:
+      logLevel:
+        description: 'Log level'
+        required: false
+        default: 'information'
+        type: choice
+        options:
+          - information
+          - debug
+          - warning
+          - critical
+      tags:
+        description: 'Purpose of Run This Workflow?'
+        required: false
+        type: string
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - name: Get old version
+        id: get_old_version
+        run: |
+          old_version=$(grep 'israeli-bank-scrapers-core@' yarn.lock | head -n1 | cut -d'@' -f2)
+          echo "::set-output name=old_version::$old_version"
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Upgrade israeli-bank-scrapers-core
+        run: |
+          yarn upgrade israeli-bank-scrapers-core --latest
+
+      - name: Get new version
+        id: get_new_version
+        run: |
+          new_version=$(grep 'israeli-bank-scrapers-core@' yarn.lock | cut -d'@' -f2)
+          echo "::set-output name=new_version::$new_version"
+
+      - name: Commit changes
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add .
+          git commit -m "chore: Upgrade israeli-bank-scrapers-core library" || echo "No changes to commit"
+
```

Comment:
You may omit this step because the `create-pull-request` action is doing it for you.

https://github.com/peter-evans/create-pull-request/tree/v6/?tab=readme-ov-file#action-behaviour

Don't forget to checkout a `branch` and delete it at the end. Read the link above carefuly.